azerothcore/azerothcore-wotlk
# fix(Scripts/quest): Improve The Force of Neltharaku

## Changes Proposed:
This PR proposes changes to:

new model structure with:
fix: duplicate Kill Credit (spell cast)
fix: added specific target condition

## Tests Performed:

This PR has been:
- [x] Tested in-game


## How to Test the Changes:


1. quest add 10854
2. go to the place that appears on the map
3. use the quest item on the drake
